server/plugin.go

```diff
@@ -259,16 +259,16 @@ func (p *Plugin) getGitlabUserInfoByMattermostID(userID string) (*gitlab.UserInf
 	config := p.getConfiguration()
 
 	var userInfo gitlab.UserInfo
-
-	if infoBytes, err := p.API.KVGet(userID + GitlabTokenKey); err != nil || infoBytes == nil {
+	var infoBytes []byte
+	if err := p.client.KV.Get(userID+GitlabTokenKey, &infoBytes); err != nil || infoBytes == nil {
 		return nil, &APIErrorResponse{ID: APIErrorIDNotConnected, Message: "Must connect user account to GitLab first.", StatusCode: http.StatusBadRequest}
 	} else if err := json.Unmarshal(infoBytes, &userInfo); err != nil {
 		return nil, &APIErrorResponse{ID: "", Message: "Unable to parse token.", StatusCode: http.StatusInternalServerError}
 	}
```

Comment:
`p.client.KV.Get` can also take of unmarshling the json payload. Let's take advantage here to simplify the code.